presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -340,6 +346,77 @@ public void testNativeMultipleInClauses()
         sqlServer.execute("SELECT count(*) FROM dbo.orders WHERE " + longInClauses);
     }
 
+    @Test
+    @Override
+    public void testShowCreateTable()
+    {
+        assertThat((String) computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .matches("CREATE TABLE \\w+\\.\\w+\\.orders \\Q(\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar(1),\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar(15),\n" +
+                        "   clerk varchar(15),\n" +
+                        "   shippriority integer,\n" +
+                        "   comment varchar(79)\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   data_compression = 'NONE'\n" +
+                        ")");
+    }
+
+    @Test(dataProvider = "dataCompression")
+    public void testCreateWithDataCompression(DataCompression dataCompression)
+    {
+        String tableName = "test_create_with_compression_" + randomTableSuffix();
+        String createQuery = format("CREATE TABLE sqlserver.dbo.%s (\n" +
+                        "   a bigint,\n" +
+                        "   b bigint\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   data_compression = '%s'\n" +
+                        ")",
+                tableName,
+                dataCompression);
+        assertUpdate(createQuery);
+
+        assertEquals(getQueryRunner().execute("SHOW CREATE TABLE " + tableName).getOnlyValue(), createQuery);
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @DataProvider
+    public Object[][] dataCompression()
+    {
+        return new Object[][] {
+                {NONE},
+                {ROW},
+                {PAGE}
+        };
+    }
+
+    @Test
+    public void testShowCreateForPartitionedTables()
```

Comment:
separate commit?

Reply:
This test was to ensure that it doesn't show data_compression for partitioned tables. Have updated the method name